formats/src/test/scala/org/scanamo/DynamoFormatTest.scala

```diff
@@ -44,4 +44,10 @@ class DynamoFormatTest extends FunSpec with Matchers with ScalaCheckDrivenProper
   testReadWrite[Set[String]](Gen.containerOf[Set, String](nonEmptyStringGen))
   testReadWrite[Option[String]](Gen.option(nonEmptyStringGen))
   testReadWrite[Option[Int]]()
+  testReadWrite[Map[String, Long]](Gen.mapOf[String, Long] {
+    for {
+      key <- nonEmptyStringGen
+      value <- Arbitrary.arbitrary[Long]
+    } yield key -> value
+  })
```

Comment:
I think you can use `Gen.zip` rather than a for comprehension.
```
Gen.mapOf[String, Long](Gen.zip(nonEmptyStringGen, Arbitrary.arbitrary[Long]))
```

Reply:
I looked for that but couldn't find it 🤔